### DISCUSSION TITLE
HERO 11 powering off

### DISCUSSION THREAD

**Discussion by jxqqk** [gopro/labs#177] [Q&A]
I have a brand new Hero 11 black, latest Lab firmware, original new battery or powered by usb.
I have a problem I am not able to resolve, I tried everything.
No matter what shutdown and delay command I send, the camera woke but after 2 sec countdown, only "powering off...".

I tried even a simple !20N:
[2023-01-05 23:12:02] Read QR:!20N 
[2023-01-05 23:12:06] Shutdown and Sleep:10 seconds 
[2023-01-05 23:12:20] wake reason:RTC_ALARM 
[2023-01-05 23:12:20] Labs:Main Start 
[2023-01-05 23:12:21] Labs:Initialize 
[2023-01-05 23:12:21] Woke with Alarm:[23:12:23]!N 
[2023-01-05 23:12:22] Woke with Command:[23:12:23]!N 
[2023-01-05 23:12:22] Start:1 seconds 
[2023-01-05 23:12:24] Timer wake command:!N 

at this point a "powering off" message.

Thank you for your help.

JM

**Comment by dnewman-gpsw** [comment]
I have been working to fix that, although if your command actually went to do something, those commands would be executed.  E.g.  !20S will start recording.

**Reply by jxqqk** [reply]
I tried !20S, if the camera wake up in photo mode it works, in video mode or timelapse mode what I have is "powering off..."

**Reply by dnewman-gpsw** [reply]
Camera in video mode start recording here.  Something is making your boot time a tad longer, more start messages maybe. This is known bug, and will be addressed in the next release.  In the meantime, try using !20SQ  or !20NQ  these are faster.  Also !MDSPL=500 for a shorter message display time on boot.

**Reply by jxqqk** [reply]
Thank you for your help.
!MDSPL=500 looks useful and now !timeSQ works in all modes, unfortunately !timeNQ still do not work.
Now maybe I can try to start a daily timelapse removing N with something like !timeSQ!timeE!1R and preselect everything on camera before start, waiting for the fixed firmware.
JM
